test/cmdlineTests/name_simplifier/output

```diff
@@ -0,0 +1,131 @@
+            }
+            function abi_decode_t_array$_t_struct$_S_$dyn(offset, end) -> array
+            {
+                if iszero(slt(add(offset, 0x1f), end)) { revert(array, array) }
+                let length := calldataload(offset)
+                if gt(length, 0xffffffffffffffff) { revert(array, array) }
+                let _1 := 0x20
+                let _2 := mul(length, _1)
+                array := allocateMemory(add(_2, _1))
+                let dst := array
+                mstore(array, length)
+                dst := add(array, _1)
+                let src := add(offset, _1)
+                if gt(add(add(offset, _2), _1), end) { revert(0, 0) }
+                let i := 0
+                let i_1 := i
+                for { } lt(i_1, length) { i_1 := add(i_1, 1) }
+                {
+                    if slt(sub(end, src), _1) { revert(i, i) }
+                    let value := allocateMemory(_1)
+                    mstore(value, calldataload(src))
+                    mstore(dst, value)
+                    dst := add(dst, _1)
+                    src := add(src, _1)
+                }
+            }
+            function abi_encode_uint256_t_string(headStart, value0, value1) -> tail
+            {
+                mstore(headStart, value0)
+                let _1 := 32
+                mstore(add(headStart, _1), 64)
+                let length := mload(value1)
+                mstore(add(headStart, 64), length)
+                let i := tail
+                for { } lt(i, length) { i := add(i, _1) }
+                {
+                    mstore(add(add(headStart, i), 96), mload(add(add(value1, i), _1)))
+                }
+                if gt(i, length)
+                {
+                    mstore(add(add(headStart, length), 96), tail)
+                }
+                tail := add(add(headStart, and(add(length, 31), not(31))), 96)
+            }
+            function allocateMemory(size) -> memPtr
+            {
+                memPtr := mload(64)
+                let newFreePtr := add(memPtr, size)
+                if or(gt(newFreePtr, 0xffffffffffffffff), lt(newFreePtr, memPtr)) { revert(0, 0) }
+                mstore(64, newFreePtr)
+            }
+            function convert_t_stringliteral_6490_to_t_string() -> converted
+            {
+                converted := allocateMemory(160)
+                mstore(converted, 100)
+                mstore(add(converted, 32), "longstringlongstringlongstringlo")
+                mstore(add(converted, 64), "ngstringlongstringlongstringlong")
+                mstore(add(converted, 96), "stringlongstringlongstringlongst")
+                mstore(add(converted, 128), "ring")
+            }
+            function extract_from_storage_value_dynamict_uint256(slot_value, offset) -> value
+            {
+                value := shr(mul(offset, 8), slot_value)
+            }
+            function memory_array_index_access_t_array$_t_struct$_S_$dyn(baseRef, index) -> addr
```

Comment:
`index_access_t_array` should be stripped here. I think rerunning the command line tests should fix it.